README.md

```diff
@@ -24,8 +24,65 @@ To cope with it, Zed's flatpak defaults can be changed to:
 When Zed's flatpak is running in the sandbox with no escape, it is not possible to execute commands on the host system.
 
 To execute commands on the host system, run inside the sandbox:
+
+```shell
+$ flatpak-spawn --host <COMMAND>
+```
+
+or
+
 ```shell
-  $ flatpak-spawn --host <COMMAND>
+$ host-spawn <COMMAND>
+```
+
+- Most users seem to report a better experience with `host-spawn`
+
+### Use host shell in the integrated terminal.
+
+Another option to execute commands is to use your host shell in the integrated terminal instead of the sandbox one.
+
+For that, open Zed's settings via <kbd>Ctrl</kbd> + <kbd>,</kbd>
+
+And make sure that you have the following lines there:
+
+`flatpak-spawn --host`
+
+```json
+{
+  "terminal": {
+    "shell": {
+      "with_arguments": {
+        "program": "/usr/bin/flatpak-spawn",
+        "args": [
+          "--host",
+          "--env=TERM=xterm-256color",
+          "sh",
+          "-c",
+          "exec $(getent passwd $USER | cut -d: -f7)"
+        ]
+      }
+    }
+  },
+}
+```
+
+`host-spawn`
+
+```json
+{
+  "terminal": {
+    "shell": {
+      "with_arguments": {
+        "program": "/app/bin/host-spawn",
+        "args": [
+          "sh",
+          "-c",
+          "exec $(getent passwd $USER | cut -d: -f7)"
```

Comment:
The example is executing the current user defined shell. Maybe stating it explicitly in the description could help people to understand it and adapt it to their needs. Is there a documentation link that could be added describing the complete syntax of that JSON file?


"The configuration syntax of these spawning commands can be found..." or similar.

And I guess the PR will be ready for a merge.

Reply:
Fixed, and thanks for the review!